utils/btrfs-progs/Makefile

```diff
@@ -6,16 +6,16 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=btrfs-progs
-PKG_VERSION:=5.1.1
+PKG_VERSION:=5.2.1
 PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-v$(PKG_VERSION).tar.xz
 PKG_SOURCE_URL:=@KERNEL/linux/kernel/people/kdave/btrfs-progs
-PKG_HASH:=9cb91b7de9e10aa6bbf2b003f60bb3f5e5b1984a8008fad7c4b2d3978f5ebe1b
+PKG_HASH:=36ac4a0198ffff79d5800c537ea4b19769a8fd3ad870f75413d25b20e2d83233
 PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-v$(PKG_VERSION)
 
 PKG_MAINTAINER:=Karel Kočí <karel.koci@nic.cz>
-PKG_LICENSE:=GPL-2.0
+PKG_LICENSE:=GPL-2.0-or-later
```

Comment:
Should be changed to -only. The source files don't mention version 2 or later. Only version 2.

Reply:
[Alpine Linux](https://pkgs.alpinelinux.org/packages?name=btrfs-progs&branch=edge&repo=main) and got a little bit confused by [repology.org](https://repology.org/project/btrfs-progs/packages), but yeah. Most kernel stuff is under GPL-2.0-only. I will change it.

Reply:
Fixed.